site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
+
+## Bug fixes and improvements
+
+After some intensive debugging by [Rodrigo](https://github.com/ReinUsesLisp), an OpenGL rendering bug in ARMS (that was with us since the game started to boot), was solved by simply updating our buildbot VM from Visual Studio 2017 to the latest 2019 version.
+Thanks to [DarkLordZach](https://github.com/DarkLordZach) for the change!
+
+PSA - Please update to or install the latest [Visual C++ 2019 x64 redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads), or else you won’t be able to open yuzu, as it will complain that the library VCRUNTIME140.dll cannot be found.
+
+{{< single-title-imgs
+    "Better, right?"
+    "./03.png"
+    "./04.png"
+  >}}
+
+perillamint and [bunnei](https://github.com/bunnei) stubbed (this means to send a fake `everything’s OK` signal so the game can continue) some VR related services.
+These changes are needed for version 1.6.0 of The Legend of Zelda: Breath of the Wild, 1.3.0 of Super Mario Odyssey, and versions 3.1.0 and above of Super Smash Bros. Ultimate.
+
+Commercial games aren't everything.
+[Rodrigo](https://github.com/ReinUsesLisp) decided to give homebrew lovers and old-school gamers something to play with, by implementing some attributes from the old OpenGL 1.X and 2.X days.
+It’s now possible to run `Xash3D`, so you can frag in Counter-Strike, or follow G-Man’s orders in Half-Life!
+Currently, this needs to be manually built with newer dependencies, but that should be solved for good in the near future.
+
+Margen67 and [Morph](https://github.com/Morph1984) wanted to improve the graphics for games and so they added Anisotropic Filtering (AF) to yuzu.
+With this feature in place, distant textures now look much sharper, improving game quality even over what the original Switch does! 
+Games like Pokémon Sword, Pokémon Shield, and Fire Emblem Three Houses benefit a lot from this.
+However, some other games don’t respond well to this new feature, one example being Astral Chain. Feel free to experiment with it!
+
+An example in Pokémon Sword:
+
+{{< imgs
+    "./15.png| AF Off"
+    "./14.png| AF 16x"
+  >}}
+  
+And another example in Fire Emblem Three Houses:  
+  
+{{< imgs
+    "./01.png| AF Off"
+    "./02.png| AF 8x"
+  >}}
+
+Shader cache invalidations are something our users don’t like at all. 
+We don't too, but it’s all part of the process of improving yuzu to achieve better performance and accuracy.
+For example, one of our recent invalidations was needed to fix the reflections in Luigi’s Mansion 3, making certain parts of the game playable.
+
+Because dancing is important, [namkazt](https://github.com/namkazt) and [Rodrigo](https://github.com/ReinUsesLisp) have been fighting with the code, allowing us to enjoy 3D graphics in Hatsune Miku: Project DIVA MEGA39's, and as collateral, fixed some rendering issues in Bayonetta 1 and Super Smash Bros. Ultimate.
```

Comment:
Just a suggestion: maybe say "because the art of dancing is important" to be more eloquent?